mobilitydb/src/point/tpoint.c

```diff
@@ -67,8 +67,8 @@
 static void
 pg_error(const char *fmt, va_list ap)
 {
-  char errmsg[PGC_ERRMSG_MAXLEN];
-  vsnprintf (errmsg, sizeof(errmsg), fmt, ap);
+  char errmsg[PGC_ERRMSG_MAXLEN + 1];
+  vsnprintf (errmsg, PGC_ERRMSG_MAXLEN, fmt, ap);
   errmsg[PGC_ERRMSG_MAXLEN]='\0';
```

Comment:
It would be better to keep `char errmsg[PGC_ERRMSG_MAXLEN];` and just remove the line with `errmsg[PGC_ERRMSG_MAXLEN]='\0';`, since vsnprintf already adds the null terminator